src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,7 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 * Vendor new SDK and bump API version to 2023-09-02-preview.
+* Add `--network-plugin` to the `az aks update` command.
```

Comment:
@zhoxing-ms , @FumingZhang - should I just bump this to 0.5.168 or does that happen elsewhere?